oeps/oep-0027-course-metadata-authoring.rst

```diff
@@ -0,0 +1,151 @@
+  synchronization issues, we will limit the editing of a piece of data to its single owning service. Anywhere else that a piece of data is editable
+  will be made read-only.
+
+* We will introduce a new front end for editing Catalog Course metadata, written using the latest recommended front end technologies and leveraging phase 3
+of the `decoupled frontend architecture`_ plan.
+
+* From the course author's perspective, we will provide a unified user experience by having the current Studio tool
+  be the portal to authoring both Course Run metadata and Catalog Course metadata. So even though authoring of Catalog Course
+  metadata will live in the Discovery subdomain, there will be a seamless transition between the front ends of Course Run
+  Authoring and Catalog Course Authoring.
+
+  * This will depend on upcoming architectural infrastructure work on single sign on across the edX microservices and a universal
+    front end header and footer.
+
+  .. _decoupled frontend architecture: https://openedx.atlassian.net/wiki/spaces/FEDX/pages/790692200/Decoupled+Frontend+Architecture
+  .. _Discovery subdomain: https://openedx.atlassian.net/wiki/spaces/AC/pages/663224968/edX+DDD+Bounded+Contexts#edXDDDBoundedContexts-edXSubdomainMap
+  .. _Content Authoring subdomain: https://openedx.atlassian.net/wiki/spaces/AC/pages/663224968/edX+DDD+Bounded+Contexts#edXDDDBoundedContexts-edXSubdomainMap
+  .. _edX subdomain: https://openedx.atlassian.net/wiki/spaces/AC/pages/663224968/edX+DDD+Bounded+Contexts#edXDDDBoundedContexts-edXSubdomainMap
+
+.. image:: oep-0027/publisher_v3_future.png
+  :alt: A flowchart of the movement of Course Run metadata and Catalog Course metadata through edX systems in the future.
+
+Consequences
+------------
+
+* By separating the concepts of Course Run metadata authoring from Catalog Course metadata authoring, we allow the overall
+  edX system to integrate better with other external systems. We have found that other systems use Open edX for the Course
+  Run learning experience and often choose to integrate with their own organizational Catalog Course management system.
+
+  As a consequence, the authoring experience is bifurcated into two different subdomains in our system, Studio and Discovery. However,
+  we do hope that the integration at the UI layer will avoid any disparate user experience. The separation of concerns in the back end
+  allows us to have clearer integration points. We believe this is a worthwhile trade-off.
+
+* By removing the workflow engine, we are moving to a model that enables self-service publishing for most courses and allows internal teams to 
+  reduce the number of blocking approvals and to move to a more scalable system of monitoring and alerting rather than manual review. There are 
+  risks associated with removing existing workflow and gates, but by offloading workflow to a specialized external system, there is a reduced 
```

Comment:
Is there a different doc that covers how the integration with external workflows will work?

Reply:
No, there isn't. I think it's too early in the process right now to iron out those details.

Reply:
You can clarify here that the workflow questions and design will be dealt with in a later document.  However, it feels like some minimal level of workflow integration should be thought through sooner rather than later to ensure the whole design holds.